phongo_compat.h

```diff
@@ -167,7 +167,10 @@
 # define Z_ISUNDEF(x) !x
 # define phongo_free_object_arg void
 # define phongo_zpp_char_len int
-# define ZEND_HASH_APPLY_COUNT(ht) (ht)->nApplyCount
+# define ZEND_HASH_APPLY_PROTECTION(ht) true
+# define ZEND_HASH_GET_APPLY_COUNT(ht) ((ht)->nApplyCount)
+# define ZEND_HASH_DEC_APPLY_COUNT(ht) ((ht)->nApplyCount -= 1)
+# define ZEND_HASH_INC_APPLY_COUNT(ht) ((ht)->nApplyCount += 1)
```

Comment:
I only see [`nApplyCount` declared in `zend_hash.h`](https://github.com/php/php-src/blob/PHP-5.6/Zend/zend_hash.h#L78) on PHP-5.6. There are no other macros that refer to it. Prior to this PR, we had a single macro to access the struct field.